style.css

```diff
@@ -164,248 +136,100 @@ p {
     width: 100%; /* Full width */
     height: 100%; /* Full height */
     overflow: auto; /* Enable scroll if needed */
-    background-color: rgb(0,0,0); /* Fallback color */
-    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
-  }
-  
-  /* Modal Content/Box */
-  .modal-content {
+    background-color: rgb(0, 0, 0); /* Fallback color */
+    background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content/Box */
+.modal-content {
     background-color: #348ea3;
-    margin: 15% auto; /* 15% from the top and centered */
     padding: 20px;
+    margin: 20px auto;
     border: 1px solid #888;
     border-radius: 50px;
     width: 40%; /* Could be more or less, depending on screen size */
-  }
-
-  #mc_embed_signup {
-      background:#348ea3;
-      clear:left;
-      font:14px var(--primary-font);
-      font-weight: 600; 
-      margin-top: 10px;
-      color: white;
-  }
-  
-  /* The Close Button */
-  .close {
+}
+
+#mc_embed_signup {
+    background: #348ea3;
+    clear: left;
```

Comment:
This is causing the `* indicates required field` to be moved to the left as well. I think it might be better if this was moved back to the right
![image](https://user-images.githubusercontent.com/6752596/86190066-66196580-bb11-11ea-9d51-44a6f0e69b8a.png)